### PR TITLE
Add list of job types and random number in variables

### PR DESCRIPTION
**Pull request by falko**

### CODE REVIEW

File: src/main/java/org/camunda/community/benchmarks/StartPiExecutor.java [hunk 1]
```diff
@@ -42,6 +41,12 @@ public class StartPiExecutor {
     public void init() throws IOException {
         String variablesJsonString = tryReadVariables(config.getPayloadPath().getInputStream());
         benchmarkPayload = zeebeClientConfiguration.getJsonMapper().fromJsonAsMap(variablesJsonString);
+        benchmarkPayload.forEach((key, value)->{
+            if(value instanceof String && !((String) value).isEmpty() && ((String) value).contains("${RANDOM_NUMBER}")) {
+                benchmarkPayload.replace(key,
+                    ((String) value).replace("${RANDOM_NUMBER}", UUID.randomUUID().toString()));
```
Comment (falko) [inline src/main/java/org/camunda/community/benchmarks/StartPiExecutor.java:47]: If it's a UUID the placeholder should probably be called `RANDOM_UUID`

File: src/main/java/org/camunda/community/benchmarks/JobWorker.java [hunk 2]
```diff
@@ -57,15 +58,30 @@ private void registerWorker(String jobType) {
...
+        String[] jobs = taskType.split(",");
         boolean startWorkers = config.isStartWorkers();
         int numberOfJobTypes = config.getMultipleJobTypes();
 
         if(startWorkers) {
             if (numberOfJobTypes <= 0) {
-                registerWorkersForTaskType(taskType);
+                if(jobs!=null && jobs.length>1) {
+                    for (int i = 0; jobs.length > i; i++) {
+                        registerWorkersForTaskType(taskType);
```
Comment (falko) [inline src/main/java/org/camunda/community/benchmarks/JobWorker.java:69]: should be changed to `jobs[i]`

File: src/main/resources/bpmn/typical_payload.json [hunk 3]
```diff
@@ -3,7 +3,7 @@
   "var2": true,
   "var3": 15,
   "var4": {
-    "var4-1": "value4-1",
+    "var4-1": "value4-1-${COUNT}",
```
Comment (falko) [inline src/main/resources/bpmn/typical_payload.json:6]: This should match the placeholder, e.g. `RANDOM_UUID`

File: src/main/java/org/camunda/community/benchmarks/StartPiExecutor.java (hunk 1 above)
Comment (plungu) [reply src/main/java/org/camunda/community/benchmarks/StartPiExecutor.java]: Changed it.

File: src/main/java/org/camunda/community/benchmarks/config/BenchmarkConfiguration.java
Comment (plungu) [inline src/main/java/org/camunda/community/benchmarks/config/BenchmarkConfiguration.java:1]: I changed the config parameter `multipleJobTypes` to `numberOfWorkersPerType`. I think it makes more sense but happy to revery it if you think it causes more trouble than good.

**Review by falko — COMMENTED** [review]
LGTM

**Pull request merged**